addon/plugins/table/nodes/table.ts

```diff
@@ -194,8 +215,26 @@ export function tableNodes(options: TableNodeOptions): TableNodes {
           },
         },
       ],
+      serialize(node, state) {
+        const pos = getPos(node, state.doc) as ResolvedPos;
+        // table rows are 1-indexed
+        const isEven = (pos.index() + 1) % 2 === 0;
+        const style = {
+          ...rowStyle,
+          ...(isEven ? evenRowStyle : oddRowStyle),
```

Comment:
When serializing this applies the correct even or odd CSS var, but we can't necessarily rely on the CSS being applied. I think for this to work as expected we'd need to apply `background: isEven ? options.rowBackground.even : options.rowBackground.odd`.

Reply:
I was thinking of e.g. when copy-pasting, we can't rely on where we're pasting to understanding the CSS variables. Actually, I just tried this in Google Docs, and while it doesn't apply the color at all with the variables approach, and if I inline the colors themselves, it does, the background is just applied to the text as a highlight, not the whole cell. Maybe this looks more broken? Then again, I think this is a feature that Google Docs doesn't have, so perhaps it's expected and might work well elsewhere?

Reply:
Sorry, that was a bit rambling...

I think the inlined version is more portable compared to the var approach when serialising. I have a slight preference for that approach, even if it might lead to odd results in some places.

Reply:
It makes sense that google docs cannot handle the variables approach, I'll also try with word inline. You're right that it's probably better to refrain from using the variables in the serialization.

Reply:
Should be changed now